README.md

```diff
@@ -7,19 +7,22 @@ Yet another F-Droid client.
 
 ## Description
 
-Unofficial F-Droid client that resembles the classic F-Droid client.
+Unofficial F-Droid client resembling the classic F-Droid client.
 
-The client supports all basic F-Droid features, such as repository management and package installation. It doesn't
-support privileged extension, root installation, or local repositories.
+Jump over the lazy dog, manage repositories and install software quickly.
+No privileged extension, root installation, or sharing local repositories nearby yet.
 
-### Features
+## Features
```

Comment:
It was intentional to use 3rd-level header here.

Reply:
I know, but the overall appearance I found unsymmetrical and noisy. I value the orderliness over the information that some things belong to each-other. That information isn't carried over well, it just appears different enough that it looks off. So it becomes a tool to remember what belongs where to the writer, which isn't needed.

Reply:
> I found unsymmetrical and noisy

I don't think so, and I prefer the old structure, this is how I structure my READMEs. Please, revert it (and on "Screenshots" block too).

Reply:
Will do.